generator/model.go

```diff
@@ -106,7 +106,7 @@ func (m *definitionGenerator) Generate() error {
 
 	mod, err := makeGenDefinition(m.Name, m.Target, m.Model, m.SpecDoc, m.opts)
 	if err != nil {
-		return err
+		return fmt.Errorf("could not generate definitions for model %s on target %s: %s", m.Name, m.Target, err.Error())
```

Comment:
instead of %s, err.Error() you can use %v, err

Reply:
Yes. Cute.